features/step_definitions/snippet_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^I am authenticated$/) do
+  user = User.new email: "landry.soules@gmail.com", password: "1Password", username: "landry"
+  signup user
+  authenticate user
+  page.assert_text "Signed in successfully."
+end
+
+When(/^I fill in the snippet creation form$/) do
+  #TODO: check why cucumber doesn't recognize routing helpers
+  #visit new_snipet
```

Comment:
Thanks a lot ! I knew there must be some way to avoid writing down the url. 
By the way, it seems that visit_new_snipet_url also works.